microservice_websocket_docker/app.py

```diff
@@ -245,6 +245,9 @@ def get_data(sensorID: str) -> dict:
     if sensor is None:
         return {}
 
+    sensor["state"] = update_state(sensor["state"], sensor["lastSeenAt"])
```

Comment:
Aggiornamento dei sensori quando si fetchano i dati. Utile si, ma se il sensore va in timeout e l'utente non aggiorna la pagina non vede risultati.